guide/additional-info/changes-in-v14.md

```diff
@@ -132,47 +135,74 @@ TypeScript users should narrow `Channel` types via type guards in order to get m
 
-### MessageComponent
+#### REST Events
 
-MessageComponents have been renamed as well. They no longer have the `Message` prefix:
+The following discord.js events:
+- `invalidRequestWarning`
+- `apiRequest`
+- `apiResponse`
+- `rateLimit`
+
+Have been removed from the `Client` in discord.js. Instead you should access these events from `Client#rest`. In addition `apiRequest`, `apiResponse` and `rateLimit` events have been renamed:
 
 ```diff
-- const button = new MessageButton();
-+ const button = new ButtonComponent();
+- client.on('apiRequest', ...);
++ client.rest.on('request', ...);
 
-- const selectMenu = new MessageSelectMenu();
-+ const selectMenu = new SelectMenuComponent();
+- client.on('apiResponse', ...);
++ client.rest.on('response', ...);
 
-- const actionRow = new MessageActionRow();
-+ const actionRow = new ActionRow();
+- client.on('rateLimit', ...);
++ client.rest.on('rateLimited', ...);
 ```
 
-Many of the analogous enums can be found be found in the discord-api-types docs (link website here)
+### RoleManager
 
-### Interaction
+`Role.comparePositions` has been removed. Use `RoleManager#comparePositions` instead.
 
-The following typeguards on `Interaction` have been renamed:
+### ThreadMemberManager
 
-```diff
-- interaction.isCommand()
-+ interaction.isChatInputCommand()
+`#fetch` now only takes a single object of type `ThreadMemberFetchOptions`.
 
-- interaction.isContextMenu()
-+ interaction.isContextMenuCommand()
-```
+### Util
 
-In addition, `Interaction#isCommand`, now indicates whether the command is an *application command* or not. This differs from the previous implementation where `isCommand()` indicated if the interaction was a chat input command or not.
+`#removeMentions` has been removed, to control mentions you should use `allowedMentions` on `MessageOptions` instead.
 
-### VoicesRegion Changes
 
-`VoiceRegion#vip` has been removed as the field is no longer part of the API.
+### `.deleted` Field(s) have been removed
 
-### Application
+You can no longer use `#deleted` to check if a structure was deleted or not. 
 
-`Application#fetchAssets` has been removed as it is no longer supported by the API.
+Check out [the issue ticket](https://github.com/discordjs/discord.js/issues/7091) for more context.
+
+### VoiceChannel
+
+`#editable` has been removed, instead you should use `GuildChannel#manageable` for checking this permission.
+
+Many of the analogous enums can be found be found in the discord-api-types docs (link website here)
+
+### VoicesRegion Changes
+
+`#vip` has been removed as the field is no longer part of the API.
 
 ## Features
 
+### Channel
+
+New typeguards have been added:
+- `#isText`*
+- `#isDM`*
+- `#isGroupDM`
+- `#isVoice`*
+- `#isVoiceBased`
+- `#isTextBased`
+- `#isStage`
+- `#isCategory`
+- `#isNews`
+- `#isStore`
+
+*These methods existed previously but have different behaviors refer to the docs for their specific changes.
```

Comment:
Iirc only `isText` and `isVoice` these 2 methods existed previously which are renamed to `isTextBased` and `isVoiceBased` so ig adding these 2 change in breaking changes is worth it.